detox/src/utils/logger.js

```diff
@@ -73,7 +73,7 @@ function createPlainBunyanStream({ logPath, level }) {
 }
 
 function init() {
-  const levelFromArg = argparse.getArgValue('loglevel');
```

Comment:
@noomorph how come this was OK for detox test, but not for other subcommands?

Reply:
Because detox test wraps -l into DETOX_LOGLEVEL environment variable for Jest, and the argparse logic works with env vars too.